sdk/spring/spring-messaging-azure-eventhubs/src/main/java/com/azure/spring/eventhubs/support/converter/EventHubsMessageConverter.java

```diff
@@ -28,11 +28,18 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(EventHubsMessageConverter.class);
 
-    private static final Set<String> SYSTEM_HEADERS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
+    private static final Set<String> IGNORED_HEADERS = Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
```

Comment:
Maybe we should improve this naming, the `IGNORED_HEADERS ` could be understood as ignoring when sending or receiving.